src/shared/components/challenge-detail/Header/index.jsx

```diff
@@ -187,12 +189,20 @@ export default function ChallengeHeader(props) {
   let nextDeadlineMsg;
   switch ((status || '').toLowerCase()) {
     case 'active':
-      nextDeadlineMsg = (
-        <div styleName="next-deadline">
-          Next Deadline: <span styleName="deadline-highlighted">
-            {nextDeadline || '-'}</span>
-        </div>
-      );
+      if (timeLeft === 0) {
```

Comment:
This does not look correct to me. To check whether a challenge is stalled, we should look at the status field in the challenge object.

Reply:
do we have a `stalled` status along with `active` and `completed` states?
where can I find a challenge in `stalled` competition to test?

Reply:
@imadx I guess so, but I might be wrong. There are many stalled F2F in the dev challenge listing, and you can find out in the listing code, what is the actual conidition to distinguish stalled challenges.

Reply:
@birdofpreyru The stalled competitions also shows their status to be `active`.
It doesn't seem to be the number of registrants either. Do you know anyone who knows what distinguishes stalled challenges?

Reply:
Ok, I checked it myself, it is all here: https://github.com/topcoder-platform/community-app/blob/develop/src/shared/components/challenge-listing/ChallengeCard/Status/index.jsx
A challenge is stalled if it has no open phases beside registration phase.

Reply:
Awesome! I'll fix this...

Reply:
@birdofpreyru made the changes and checked with the challenge listing.
I hope the behavior is fixed now.